File: C_Programming/Practice/program5.c
#include<stdio.h>

int main()
{
    float fValue1 = 0.0f;                               //To store first input
    float fValue2 = 0.0f;                               //To store second input
    float fResult = 0.0f;                               //To store result

    printf("Enter the first number : \n");
    scanf("%f",&fValue1);
    printf("Enter the second number : \n");
    scanf("%f",&fValue2);

    fResult = fValue1 + fValue2;                        // perform the Addtion and it is Business logic

    printf("Addition is : %f",fResult);
    

    return 0;
}

File: Pyhton/Competitive/Assignment9/Program5.py
def ChkDivisible(a):
    if(a  % 3 == 0 and a % 5 == 0):
        print(a, " is  Divisible by 3 and 5")
    else:
        print(a, " is not Divisible by 3 and 5")

a = int(input("Enter the number : "))

ChkDivisible(a)